nextflow.config

```diff
@@ -1,133 +1,168 @@
 /*
  * -------------------------------------------------
- *  nf-core/bacterialmappingphylogeny Nextflow config file
+ *  nf-core/bactmap Nextflow config file
  * -------------------------------------------------
  * Default config options for all environments.
- * Cluster-specific config options should be saved
- * in the conf folder and imported under a profile
- * name here.
  */
 
 // Global default params, used in configs
 params {
 
-  // Container slug. Stable releases should specify release tag!
-  //   Developmental code should specify :latest
-  container = 'nfcore/bacterialmappingphylogeny:latest'
+  // Input options
+  input                      = ''
 
-  // Workflow flags
-  // TODO nf-core: Specify your pipeline's command line flags
-  reads = "data/*{1,2}.fastq.gz"
-  singleEnd = false
-  outdir = './results'
+  // Reference
+  reference                  = ''
+
+  // ******************* OPTIONAL PROCESSES ******************* //
+  // Trimming
+  trim                       = true
+  save_trimmed_fail          = false
+
+  //When downsampling specifiy the maximum coverage depth
+  subsampling_off            = false
+  subsampling_depth_cutoff   = 100
+
+  // When downsampling and all samples have the same expected genome size specify as e.g 210820b or 2.1mb
+  genome_size                = false 
+
+  // Remove recombination using gubbins prior to building tree
+  remove_recombination       = false
+
+  // Which tree building alogorithms to use
+  rapidnj                    = false
+  fasttree                   = false
+  iqtree                     = false
+  raxmlng                    = false
 
   // Boilerplate options
-  name = false
-  multiqc_config = "$baseDir/conf/multiqc_config.yaml"
-  email = false
-  plaintext_email = false
-  help = false
-  igenomes_base = "./iGenomes"
-  tracedir = "${params.outdir}/pipeline_info"
-  clusterOptions = false
-  awsqueue = false
-  awsregion = 'eu-west-1'
+  outdir                            = './results'
+  tracedir                          = "${params.outdir}/pipeline_info"
+  publish_dir_mode                  = 'copy'
+  email                             = ''
+  email_on_fail                     = ''
+  plaintext_email                   = false
+  max_multiqc_email_size            = '25.MB'
+  skip_multiqc                      = false
```

Comment:
```suggestion
  skip_multiqc                      = false
  enable_conda                    = false
```

Reply:
That should remove the Warning in the CI: `WARN: access to undefined parameters...`.
You might need to add this param to the json schema as well by running once more:
`nf-core schema build .`